cmd/loadtest/account.go

```diff
@@ -767,6 +790,17 @@ func (ap *AccountPool) Next(ctx context.Context) (Account, error) {
 
 // Checks multiple conditions of the account and funds it if needed
 func (ap *AccountPool) fundAccountIfNeeded(ctx context.Context, account Account, forcedNonce *uint64, waitToFund bool) (*types.Transaction, error) {
+	// If funding amount is zero, skip funding entirely
+	if ap.fundingAmount.Cmp(big.NewInt(0)) == 0 {
+		// Log only once per account pool
+		if !ap.fundingDisabledLogged {
+			log.Info().
+				Msg("Funding disabled - skipping account funding for all accounts")
+			ap.fundingDisabledLogged = true
+		}
+		return nil, nil
```

Comment:
imho this message should be logged in debug mode and maybe I'm missing something but what's the point of having this new `fundingDisabledLogged` parameter?